### PR TITLE
Add stable diffusion notebooks to community model garden

### PR DESCRIPTION
**Pull request by xiangxu-google**
**REQUIRED:**
- Add model_garden_pytorch_stable_diffusion.ipynb for stable-diffusion-v1.5 model card.
- Add model_garden_pytorch_stable_diffusion_inpainting.ipynb for stable-diffusion-inpainting model card.

**REQUIRED:** Fill out the below checklists or remove if irrelevant

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ Y] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ Y] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb [hunk 1]
```diff
@@ -0,0 +1,702 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>H1 heading move to first line in links cell below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>Vertex-AI -&gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>GCS -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>Default the region to "us-central1"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>I assume the training container has the training script and model? Its not obvious.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: <p>ttps://drive.google.com/drive/folders/1BO_dyz-p65qhBRRMRA4TbZ8qW4rB99JZ -&gt; use bucket GCS bucket</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb [hunk 2]
```diff
@@ -0,0 +1,625 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: <p>What's the correct branding? Vertex Model Garden or Vertex AI Model Garden?</p><p> </p><p>Move H1 heading to first line in next links cell.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: <p>GCS -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: <p>use default for REGION -&gt; us-central1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: <p>Explain what is in the training container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: <p>Use public Cloud Storage bucket, not Drive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb (hunk 1 above)
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: Added more clarification.

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:1]: This google drive is a public drive created by OSS community, we don't want to host them in our GCS bucket.
See https://huggingface.co/docs/diffusers/training/dreambooth#dog-toy-example

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb (hunk 2 above)
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: Vertex AI Model Garden is the correct one, updated.
This notebook is created in Vertex AI workbench, where we will point our users to launch the notebook. So all cells and heading positions are arranged by the workbench notebook metadata.

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb:1]: done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**